.taskcluster.yml

```diff
@@ -7,6 +7,7 @@ tasks:
     - $let:
           trustDomain: mobile
           canonicalRepo: git@github.com:mozilla-mobile/firefox-android.git
+          canonicalHttpsRepo: https://github.com/mozilla-mobile/firefox-android
```

Comment:
Follow-up: Damn! I'm sad we now have to support both SSH and HTTP urls in `.taskcluster.yml`. It seems We're still getting SSH urls from the Github Webhook. I'd like switch back to HTTP, just like the rest of our Github repos. I didn't see any settings I can use to make the switch.

@csadilek, do you have the rights to revoke the firefoxci-taskcluster Github App[1] then to install it again? I wonder if that resets the URL to HTTP now that the repo is public. 🤔 

[1] https://github.com/mozilla-mobile/firefox-android/settings/installations
[2] https://github.com/apps/firefoxci-taskcluster

Reply:
+1 : It would be tidier and less error prone if we used HTTP urls everywhere.

Reply:
@JohanLorenzo Yes, I do. I can grant you that permission as well, but otherwise, happy to try.